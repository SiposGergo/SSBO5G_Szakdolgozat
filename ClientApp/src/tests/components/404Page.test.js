import React from "react";
import { shallow } from "enzyme";
import ErrorPage from "../../components/404Page";

test('Should render 404 page correctly', () => {
    const wrapper = shallow(<ErrorPage />);
    expect(wrapper).toMatchSnapshot();
});

