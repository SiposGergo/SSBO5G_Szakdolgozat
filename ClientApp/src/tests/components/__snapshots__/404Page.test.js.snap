// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render 404 page correctly 1`] = `
<div>
  A kért oldal nem található!
</div>
`;
